plugin/src/test/java/io/jenkins/plugins/analysis/warnings/recorder/JobActionITest.java

```diff
@@ -48,18 +57,22 @@ public void shouldShowTrendChart() {
         Run<?, ?> build = buildWithResult(project, Result.SUCCESS);
         assertActionProperties(project, build);
 
-        HtmlPage jobPage = getWebPage(JavaScriptSupport.JS_DISABLED, project);
-        assertThatTrendChartIsHidden(jobPage); // trend chart requires at least two builds
+        project.getActions(JobAction.class);
+        List<JobAction> jobActions = project.getActions(JobAction.class);
 
-        assertThatSidebarLinkIsVisibleAndOpensLatestResults(jobPage, build);
+        assertThatTrendChartIsHidden(jobActions.get(0)); // trend chart requires at least two builds
+
+        //TODO This should be done too. Not sure how to check sidebar links without UI
+        //assertThatSidebarLinkIsVisibleAndOpensLatestResults(jobPage, build);
```

Comment:
For each of the job actions:
- verify that getUrlName is e.g. "eclipse"
- verify that getIonName is e.g. "xyz"